### PR TITLE
Entries map

### PR DESCRIPTION
**Pull request by NadineGrosskreuz**
Self-created entries and static data are displayed on the map with a marker
Styling of the map is not finished!
User Story #24

### CODE REVIEW

File: src/components/Form/Form.js [hunk 1]
```diff
@@ -37,12 +39,13 @@ export default function Form({ id }) {
...
 		if (entryToUpdate) {
 			controlEntry(id, data);
 			editEntry(id);
 		} else {
-			addEntry(data);
+			const geoData = await fetchAddressData(watch('address'));
+			addEntry({ ...data, position: [Number(geoData[0].lat), Number(geoData[0].lon)] });
 			event.target.reset();
 			modalShow();
 		}
```
Comment (UrsulaMontag) [inline src/components/Form/Form.js:51]: Maybe you want your user see the made entry after submit? Actually, the user stay on the create-page after submitting the form. You can use router to push the user either to the map or to the list page.

**Review by UrsulaMontag — COMMENTED** [review]
Looks very nice. 😍
Think about my suggestions before the merge.

**Review by UrsulaMontag — APPROVED** [review]

File: src/components/Form/Form.js (hunk 1 above)
Comment (NadineGrosskreuz) [reply src/components/Form/Form.js:51]: I have already mentioned this in one of my next user stories, thanks! :)

File: package.json [hunk 2]
```diff
@@ -7,8 +7,8 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
-    "prettier:check": "npx prettier --check .",
-    "prettier:write": "npx prettier --write .",
+    "prettier:check": "npx prettier --ignore-unknown --check .",
+    "prettier:write": "npx prettier --ignore-unknown --write .",
     "eslint": "npx eslint \"**/*.js\"",
     "prepare": "husky install",
     "storybook": "start-storybook -p 6006 --no-manager-cache",
```
[the remark this answers is not in the document]
Comment (NadineGrosskreuz) [reply package.json:14]: Yes, Alex told me to do so because of the marker PNGs. I do not know which is the best solution.

**Pull request merged**